alignak/daemon.py

```diff
@@ -229,6 +231,7 @@ def __init__(self, name, config_file, is_daemon, do_replace, debug, debug_file):
         """
         self.check_shm()
 
+        self.daemon_type = 'unknown'
```

Comment:
Why not define the 'unknown' in the default value in the `properties` dictionary ?